web/src/features/hfiCalculator/util.ts

```diff
@@ -19,7 +19,21 @@ export const getZoneFromAreaName = (areaName: string): string => {
   return areaName.slice(-3)
 }
 
+export const getDailiesForCSV = (
+  numPrepDays: number,
+  dailies: StationDaily[]
+): StationDaily[] => {
+  // Group all dailies by their station code, then take only the number we need for each station
```

Comment:
Thank you for this comment to explain what's going on - all the chained functions are a lot to take in otherwise

Reply:
Yeah they can be terse -- but they prevent a lot of bugs if I were to write all this out imperatively.